lib/hexpm_web/templates/dashboard/organization/index.html.eex

```diff
@@ -128,7 +128,7 @@
                   <%= form_tag Routes.organization_path(Endpoint, :update, @organization), class: "remove-role" do %>
                     <input type="hidden" name="action" value="remove_member">
                     <input type="hidden" name="organization_user[username]" value="<%= organization_user.user.username %>">
-                    <%= submit "Remove", class: "btn btn-danger", disabled: organization_user.user.id == @current_user.id %>
```

Comment:
It can be an easy to miss click on yourself which is an irreversible action. What do you think about instead adding a "Danger zone" panel where we have a button to leave the organization? In the future we can also add a button to delete the organization in this panel.

Reply:
Seems to be a good approach. @ericmj Do you think would make sense in that danger zone require some kind of confirmation? Like type the name of the user/organization or type `confirm`?

Reply:
Typing in the name of organization would be a great extra safety 👍

Reply:
@ericmj Just added the "Danger zone". WDYT?